src/dc_chat.rs

```diff
@@ -776,14 +776,15 @@ unsafe fn get_parent_mime_headers(
         || parent_in_reply_to.is_null()
         || parent_references.is_null())
     {
+        // prefer a last message that isn't from us
         success = (*chat)
             .context
             .sql
             .query_row(
                 "SELECT rfc724_mid, mime_in_reply_to, mime_references \
-                 FROM msgs WHERE timestamp=(SELECT max(timestamp) \
+                 FROM msgs WHERE chat_id=? AND timestamp=(SELECT max(timestamp) \
                  FROM msgs WHERE chat_id=? AND from_id!=?);",
-                params![(*chat).id as i32, 1],
+                params![(*chat).id as i32, (*chat).id as i32, DC_CONTACT_ID_SELF],
```

Comment:
You can use `?1' and `?2' to avoid passing chat ID twice.
Or even use [`named_params!`](https://docs.rs/rusqlite/0.20.0/rusqlite/macro.named_params.html) macro.

Reply:
unfortunately the `sql.rs` wrapper thing doesn't support/ exposes the query_row_named function yet..

Reply:
You can still use `?1` and `?2` as in the example on https://docs.rs/rusqlite/0.20.0/rusqlite/